### PR TITLE
Dutch Translation updates.

### PR DESCRIPTION
**Pull request by jjbennett**
# Critical Changes

# Changes

# Issues Closed

### CODE REVIEW

File: src/translations/nl_NL.translation [hunk 1]
```diff
@@ -1,87 +1,89 @@
...
         <name>DefaultContactTransactionCurrency</name>
     </customLabels>
     <customLabels>
-        <label><!-- Suffix added to the name of the original donation to form matching donation name --></label>
+        <label>{3} Matching Donatie {1}</label>
         <name>Matching_Donation_Record_Suffix</name>
     </customLabels>
     <customLabels>
-        <label><!-- Record type for matching donation. Set to $Source to match the original donation --></label>
+        <label>$Bron</label>
```
Comment (force2b) [inline src/translations/nl_NL.translation]: This doesn't look like a record type

File: src/translations/nl_NL.translation [hunk 2]
```diff
@@ -1,87 +1,89 @@
...
+        <label>Het is mogelijk om een Opportunity Contactpersoon Rol automatisch aan te maken bij het creëeren van Opportunities. Er is een veld op Opportunity genaamd &quot;Contactpersoon Id voor Rol&quot;. Wanneer Opportunities geïmporteerd met een geldig Contactpersoon Id in dat veld, zal een Contactpersoon Rol gemaakt worden volgens de configuratie hieronder. Deze functionaliteit kan volledig uitgeschakeld worden door het wijzigen van de Trigger Configuratie.</label>
         <name>Contacts_Settings_Contact_Roles_Help_Text</name>
     </customLabels>
     <customLabels>
         <label><!-- DefaultContactTransactionCurrency --></label>
         <name>DefaultContactTransactionCurrency</name>
     </customLabels>
     <customLabels>
-        <label><!-- Suffix added to the name of the original donation to form matching donation name --></label>
+        <label>{3} Matching Donatie {1}</label>
```
Comment (force2b) [inline src/translations/nl_NL.translation]: Should these be merge fields?

Comment (jjbennett) [reply src/translations/nl_NL.translation]: This one looks correct, in English we have "{3} Matching Donation {1}" as the label.

File: src/translations/nl_NL.translation (hunk 1 above)
Comment (jjbennett) [reply src/translations/nl_NL.translation]: Great catch @force2b it looks like we are using this in the "OPP_MatchingDonationsBTN_CTRL" - so it should not be translated. It looks like German also has it translated so I will set those both back to the English $Source and add this to the do not translate list.

**Pull request merged**